.test-infra/jenkins/PreCommit_Pipeline.groovy

```diff
@@ -0,0 +1,73 @@
+#!groovy
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import hudson.model.Result
+
+List<Object> ghprbArgs = [
```

Comment:
Is there a special reason for the massive abbreviation? I _think_ this stands for "**G**it**H**ub **P**ull **R**equest **B**uild **A**rgs". I see maybe why you need a unique prefix in the XML but beyond that I'd just call it `buildArgs` or something, or if you leave the name cryptic, comment it.